.mailmap

```diff
@@ -3,3 +3,4 @@ James Braza <james@futurehouse.org> <jamesbraza@gmail.com>
 Michael Skarlinski <mskarlinski@futurehouse.org> mskarlin <12701035+mskarlin@users.noreply.github.com>
 Ryan-Rhys Griffiths <ryan@futurehouse.org> <ryangriff123@gmail.com>
 Siddharth Narayanan <sid@futurehouse.org> <sidnarayanan@users.noreply.github.com>
+Ludovico Mitchener <ludo@futurehouse.org> ludomitch <ludo@futurehouse.org>
```

Comment:
```suggestion
Ludovico Mitchener <ludo@futurehouse.org> ludomitch
```

I think if you run `git config` on your local machine, you won't need to do this